packages/svelte/src/performance.ts

```diff
@@ -1,7 +1,6 @@
 import { getCurrentScope } from '@sentry/browser';
 import type { Span, Transaction } from '@sentry/types';
 import { afterUpdate, beforeUpdate, onMount } from 'svelte';
-import { current_component } from 'svelte/internal';
```

Comment:
Can we not conditionally access this via a version check?

In `SvelteKit` we can inject stuff via vite plugin so we can inject a version into the bundle in if Svelte themselves does not expose it. Or is that problematic for other reasons?